lib/algoliasearch-rails.rb

```diff
@@ -225,28 +225,29 @@ def to_settings
         v = get_setting(k)
         settings[k] = v if !v.nil?
       end
-      settings[:slaves] = additional_indexes.select { |options, s| options[:slave] }.map do |options, s|
+      settings[:replicas] = additional_indexes.select { |options, s| options[:slave] || options[:replica] }.map do |options, s|
         name = options[:index_name]
         name = "#{name}_#{Rails.env.to_s}" if options[:per_environment]
         name
-      end if !@options[:slave]
+      end if !@options[:slave] && !@options[:replica]
       settings
     end
 
     def add_index(index_name, options = {}, &block)
-      raise ArgumentError.new('Cannot specify additional index on a slave index') if @options[:slave]
+      raise ArgumentError.new('Cannot specify additional index on a replica index') if @options[:slave] || @options[:replica]
       raise ArgumentError.new('No block given') if !block_given?
       raise ArgumentError.new('Options auto_index and auto_remove cannot be set on nested indexes') if options[:auto_index] || options[:auto_remove]
       options[:index_name] = index_name
       @additional_indexes ||= {}
       @additional_indexes[options] = IndexSettings.new(options, Proc.new)
     end
 
-    def add_slave(index_name, options = {}, &block)
-      raise ArgumentError.new('Cannot specify additional slaves on a slave index') if @options[:slave]
+    def add_replica(index_name, options = {}, &block)
+      raise ArgumentError.new('Cannot specify additional replicas on a replica index') if @options[:slave] || @options[:replica]
       raise ArgumentError.new('No block given') if !block_given?
-      add_index(index_name, options.merge({ :slave => true }), &block)
+      add_index(index_name, options.merge({ :replica => true }), &block)
     end
+    alias :add_slave :add_replica
```

Comment:
This is not the way we went so far; I don't think we want to print something in the logs for instance.